A/Atomsk/build_tarballs.jl

```diff
@@ -3,29 +3,45 @@
 using BinaryBuilder, Pkg
 
 name = "Atomsk"
-version = v"0.11.2"
+version = v"0.12.0"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/pierrehirel/atomsk.git", "3333858281e0ebd6279825b83ab871cf4d050a8d")
+    GitSource("https://github.com/byu-cxi/atomsk.git", "adeb5d5c059a16bf649ab6657cc550b5868fac0b"),
+    DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/atomsk/src/
-mkdir -p ${bindir}
+cd $WORKSPACE/srcdir/atomsk
+if [[ "$nbits" == 32 || "$target" == *apple* ]]; then
+    atomic_patch -p1 ../patches/atomsk_32.patch
+else
+    atomic_patch -p1 ../patches/atomsk_64.patch
+fi
+if [[ "$target" == *mingw* ]]; then
+    export LDFLAGS="-L$bindir"
+fi
+cd src
+
 # The makefile doesn't handle parallel builds
-make atomsk BIN="atomsk${exeext}"
-make install INSTPATH=${prefix} BIN="atomsk${exeext}"
+if [[ ! -d "$bindir" ]]; then
+    mkdir ${bindir}
+fi
```

Comment:
```suggestion
mkdir -p ${bindir}
```
but I find it bizarre the build system isn't able to deal with this

Reply:
I don't know why, but it seems that the `bindir` directory isn't there for all but the windows os.